### ISSUE TITLE
Some questions about training

### ISSUE THREAD

**Issue by xz-123-new** [SherlockHolmes221/DOQ#5]

**Comment by xz-123-new** [comment]
Sorry to interrupt again,i have some questions about the training process,hoping to hear your suggestions.                                      
 I am wondering how long it cost you to train this model basing on 8 GPUs, and the epoch on your paper is 80 while in the code its 150 in default,should i change it to 80 since i found it costful to train this model?                                                                                      Besides,i encounter a problem that when i set nproc_per_node =4,it train with 8 GPUS,and if i set it as 8 ,it will stuck and can't train,which really confuses me.How could i fix this problem.Thanks for your generous reply!

**Comment by SherlockHolmes221** [comment]
yes,  change it to 80, sorry, i forgot update the training script

set nproc_per_node as 4,it should be trained with 4 GPUS， could provide mode details

**Comment by xz-123-new** [comment]
Thanks! Besides, when i train this model on hicodet ,it will cost me an hour for each epoch on 4 P40 GPUs, is it normal to cost so many time ?

**Comment by SherlockHolmes221** [comment]
1.5days on 8 Titanx GPUs, may be can be a reference

**Comment by xz-123-new** [comment]
Could you please offer the final trained weights of this model? Because i found it hard to converge during my own training process,thanks!

**Comment by SherlockHolmes221** [comment]
https://drive.google.com/drive/folders/1zHOOI-60dazyCSGsE0cqccWQ_fhNARgH
(tips:the lr should be changed according to the bs)

**Comment by xz-123-new** [comment]
Thanks for your generous help and marvelous work!

**Comment by xz-123-new** [comment]
Have you done any ablation study on the Loss coefficients? Because i found the loss coefficients for verb,object, regression and GIoU in your code is 1,1,5,2 seperately, which is different from previous work such as qpic.And the coefficients for the matching loss is set as 1,could it be better if i change it? Thanks!

**Comment by xz-123-new** [comment]
Have you done any ablation study on the loss coefficients?Because the coefficients for match loss and match attention loss is set as 1 in default, i wonder could it be better if i change it? Thanks!

**Comment by SherlockHolmes221** [comment]
yes, we have tired some loss coefficients, and found set as 1 is best.
you can also do some ablation study on the loss coefficients.

**Comment by xz-123-new** [comment]
Could you please share the final weights with backbone resnet101 on hicodet or the pretrained model detr-r101-pre.pth?Because without pretrained model with resnet101 as backbone, i found it troublesome to train it,really thanks and sorry to trouble you again.

**Comment by SherlockHolmes221** [comment]
python convert_parameters.py  --load_path **detr-r101.pth**  --save_path params/detr-r101-pre-hico.pth

detr-r101.pth is from [detr](https://dl.fbaipublicfiles.com/detr/detr-r101-2c7b67e5.pth)

**Comment by xz-123-new** [comment]
Thanks!
